var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -109,3 +115,9 @@ def install(self, spec, prefix):
         env['PWD'] = os.getcwd()
         make()
         make('install')
+        if spec.satisfies('+setgid'):
```

Comment:
@ohm314 : I would prefer to do this post-install hook:

```
    @run_after('install')
    def change_group(self):
```

Instead of joining path may be better to search binary ? see in tau package:

```
    def get_makefiles(self):
        pattern = join_path(self.prefix.lib, 'Makefile.*')
        return glob.glob(pattern)
```

Instead of conflict, may be we should just `pass` if the `likwid-accessD` doesn't exist in above post install hook.

Reply:
OK, I will implement the first two of your suggestions, make sense. but the pass I don't like so much because i prefer to fail if one feature is not available. `likwid-accessD` was only introduced in 4.1

Reply:
ok, agree.